### PR TITLE
[JENKINS-26918] Workflow support

### PR DESCRIPTION
**Pull request by jglick**
[JENKINS-26918](https://issues.jenkins-ci.org/browse/JENKINS-26918)

Pretty simple, just binds the `manager` global variable to the existing functionality.

@reviewbybees

### CODE REVIEW

**Comment by abayer** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -64,11 +64,30 @@
...
     <dependencies>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
-            <version>1.5</version>
+            <version>1.15</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-project</artifactId>
```
Comment (amuniz) [inline pom.xml]: Why is this needed?

Comment (jglick) [inline pom.xml]: Depends on this plugin’s APIs, and it was split from core between then and now.

**Comment by amuniz** [comment]
:bee:

**Comment by ikedam** [comment]
+1

Notes about this change
* This change breaks binary compatibilities (by changing types of arguments and return values).
    * `BadgeManager` is used in groovy scripts and breaking binary compatibilities is considered harmless.
    * To keep compatibilities require new methods replacing `getBuild` and `getListener`, which should be confusing for almost users.
* `CpsScript.$build()` requires workflow-plugin 1.11, which requires Jenkins 1.609.1.

**Comment by ikedam** [comment]
Is this still under review by @reviewbybees ?
(as far as I know, @reviewbybees posts a comment when finishing the review)

**Comment by recena** [comment]
@ikedam you can consider this PR reviewed by the bees.

**Comment by ikedam** [comment]
@recena Thanks.
I'll make a new release this weekend.
